src/cycax/cycad/cycad_side.py

```diff
@@ -82,6 +82,7 @@ def hole(
         self,
         pos: tuple[float, float],
         diameter: float,
+        sink: float = 0.0,
```

Comment:
awesome. Thanks. I am laughing at my self now. By adding sink, it is not a hole anymore it should be a cylinder ... but I would hate that.